models/cow_protocol/tx_hash_labels/early_investment/ethereum/cow_protocol_tx_hash_labels_early_investment_ethereum.sql

```diff
@@ -0,0 +1,54 @@
+{{
+    config(
+        alias='tx_hash_labels_early_investment_ethereum',
+    )
+}}
+
+with
+  project_starts as (
+    select
+        token_bought_address,
+        min(block_date) as project_start
+    from (
+        select token_bought_address, block_date
+        from {{ ref('dex_aggregator_trades') }}
+        where blockchain = 'ethereum'
+        UNION ALL
+        select token_bought_address, block_date
+        from {{ ref('dex_trades') }}
+        where blockchain = 'ethereum'
+    )
+    group by
+        token_bought_address
+
+ ),
+
+ early_investment_trades as (
+    select
+        tx_hash
+    from (
+        select tx_hash, block_date, token_bought_address
+        from {{ ref('dex_aggregator_trades') }}
+        where blockchain = 'ethereum'
```

Comment:
Any reason to restrict to Ethereum here?

Reply:
We're currently focused solely on Ethereum and are the only ones using it. Do you see a need to expand this?